lambda/invoke_loop.go

```diff
@@ -100,10 +99,13 @@ func convertInvokeRequest(invoke *invoke) (*messages.InvokeRequest, error) {
 	return res, nil
 }
 
-func safeMarshal(v interface{}) []byte {
+// mustMarshal is like json.Marshal but panics if the v cannot be marshaled.
+// use this if you know the v doesn't contain values that json.Marshal can't marshal,
+// such as Channel, complex, and function values.
+func mustMarshal(v interface{}) []byte {
 	payload, err := json.Marshal(v)
 	if err != nil {
-		return []byte(fmt.Sprintf(serializationErrorFormat, err.Error()))
+		panic("lambda: " + err.Error())
```

Comment:
Sorry, I've misunderstood.
`json.Marshal(v)` doesn't return any error, because we know `*messages.InvokeResponse_Error` can be marshaled.
So, we don't need to treat this case, and just enough to panic.

Reply:
"Must"-prefix is usually used in Go (e.g. https://golang.org/pkg/regexp/#MustCompile https://golang.org/pkg/html/template/#Must )